core/.env_sample

```diff
@@ -1,2 +1,7 @@
-MNEMONIC="your mnemonic here"
-SLACK_WEBHOOK="web hook for your slack app here"
\ No newline at end of file
+MNEMONIC=""           # truffle wallet mnemonic. Used in all scripts, tools & cli.
+SLACK_WEBHOOK=""      # OPTIONAL: webhook to send messages to a slack channel. See Logger.js
+TWILIO_SID=""         # OPTIONAL: From twilio. Acts like an account ID.
+TWILIO_AUTH=""        # OPTIONAL: TWILIO API key.
+TWILIO_FROM_NUMBER="" # OPTIONAL: Caller number that originates the calls from twilio.
+DRI_NUMBER1=""        # OPTIONAL: First recipient number including international code.
+DRI_NUMBER2=""        # OPTIONAL: Second number. any n # of phone numbers can be added.
```

Comment:
Which of these are required for twilio to work? It might make sense to add sections in this file so it's clear what variables are required to enable certain features?

Reply:
to get twilio to work you need all of them. the only optional part is how many additional phone numbers you add over and above `DRI_NUMBER1`